service/src/main/java/uk/gov/hmcts/reform/fnp/exception/FeeRegisterException.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.fnp.exception;
+
+public class FeeRegisterException extends RuntimeException {
+
+    private final int status;
+
+    public FeeRegisterException() {
+        super();
+        this.status =  -1;
+    }
+
+    public FeeRegisterException(int status) {
+        super();
+        this.status = status;
+    }
+
+    public FeeRegisterException(int status, String message) {
+        super(message);
+        this.status = status;
+    }
+
+    public FeeRegisterException(int status, String message, Throwable cause) {
```

Comment:
This is the only constructor used, do we need others?

Reply:
Was just overloading the default exception methods, can remove the others if you want

Reply:
I'd remove that if it's not used.